mlflow/utils/rest_utils.py

```diff
@@ -105,17 +109,21 @@ def http_request(
             # Databricks SDK `APIClient.do` API is for making request using
             # HTTP
             # https://github.com/databricks/databricks-sdk-py/blob/a714146d9c155dd1e3567475be78623f72028ee0/databricks/sdk/core.py#L134
-            raw_response = ws_client.api_client.do(
-                method=method,
-                path=endpoint,
-                headers=extra_headers,
-                raw=True,
-                query=kwargs.get("params"),
-                body=kwargs.get("json"),
-                files=kwargs.get("files"),
-                data=kwargs.get("data"),
-            )
-            return raw_response["contents"]._response
+            with warnings.catch_warnings():
+                warnings.filterwarnings(
+                    "ignore", message=f".*{_DATABRICKS_SDK_RETRY_AFTER_SECS_DEPRECATION_WARNING}.*"
```

Comment:
is there a reason you don't hardcode the message?

Reply:
Oh cuz I think it's a bit long 😅

Reply:
I see, it's used in `tests/utils/test_rest_utils.py`.